plugin/evm/orderbook/hubbleutils/config.go

```diff
@@ -3,14 +3,25 @@ package hubbleutils
 var (
 	ChainId           int64
 	VerifyingContract string
-	HState            *HubbleState
+	hState            *HubbleState
 )
 
 func SetChainIdAndVerifyingSignedOrdersContract(chainId int64, verifyingContract string) {
 	ChainId = chainId
 	VerifyingContract = verifyingContract
 }
 
-func SetHubbleState(hState *HubbleState) {
-	HState = hState
+func SetHubbleState(_hState *HubbleState) {
+	hState = _hState
+}
+
+func GetHubbleState() *HubbleState {
+	return &HubbleState{
+		Assets:             hState.Assets,
```

Comment:
this should return a deepcopy of hState, so that the values can't be modified accidentally, and all updates go through SetHubbleState

Reply:
haha i was lazy. done now.